dashboards/jenkins/jenkins-prometheus.json

```diff
@@ -137,7 +155,7 @@
           "scorecard": {
             "blankView": {},
             "timeSeriesQuery": {
-              "prometheusQuery": "jenkins_executor_in_use_value"
+              "prometheusQuery": "jenkins_executor_in_use_value{${Cluster},${Location},${Namespace}}"
```

Comment:
The scorecards and gauges here might run into the same issue as https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/380#discussion_r1046173234